lambda-utils/lib/lambda-utils.ts

```diff
@@ -8,6 +8,21 @@ const logger = new Logger('lambda-utils');
 /** Define the async version of ProxyHandler */
 export type AsyncProxyHandler = (event: APIGatewayEvent, context: Context) => Promise<any>;
 
+/**
+ * Casted interface for APIGatewayProxyEvents converted through the middleware
+ *
+ * @typedef {Interface}
+ * @property {any} body The cast JSON -> object body element
+ */
+export interface ISailplaneAPIGatewayProxyEvent extends APIGatewayProxyEvent {
```

Comment:
Unless you work at Microsoft, let's not use Hungarian notation. 😜

I'm tempted to call this APIGatewayProxyEvent still. Since LambdaUtils is used as `import * as LambdaUtils from "@sailplane/lambda-utils"`, it would become `LambdaUtils.APIGatewayProxyEvent` to distinguish it.

In order to do this declaration, you add ` as AWS_APIGatewayProxyEvent` to the import, and then use that name here.

Reply:
One of the linter rules I have states to keep Interface prefixed with an "I". Is that not the case in normal practice? I'm hoping not because I'd love to turn that rule off

Reply:
I've never seen that rule. Turn it off.
https://github.com/palantir/tslint/issues/1520

Reply:
^ That's the rule I'm referring to :)